src/sass/user-profile.scss

```diff
@@ -0,0 +1,26 @@
+// Variables
+
+@import 'style';
+
+$button-stroke: inset 0 0 0 1px;
+
+@media screen and (min-width: 600px) {
+  form {
+    max-width: 32em;
+  }
+}
+
+// TODO: Look into why svgs aren't rendering
```

Comment:
I have this same problem! It's affecting the dropdown arrow in the select boxes as well. I'm not sure where to start looking but maybe something is happening to the svgs during the build process? The svg file itself renders fine in the browser